pkg/client/metrics/metrics.go

```diff
@@ -45,8 +45,47 @@ var (
 		},
 		[]string{"path", "server"},
```

Comment:
Huh, if there are existing metrics, what port are these served from?

Reply:
Did a bit of a deep dive to understand what's going on here. Most of the knowledge comes from ChatGPT.

The metric above is registered not to prometheus, but to the K8s APIs. Instead of being exposed normally through a port like prometheus metrics, it seems to be exposed on the default secure port of the pod (6443). 

More in depth:
We register a metric for the queryLatency to the K8s Resource API, which is used by things like HPA. 
We can't/shouldn't expose metrics to prometheus from the same port, and the handler used to register the queryLatency is not the same as the handler used to register metrics to prometheus.